src/mysql.erl

```diff
@@ -545,15 +574,20 @@ in_transaction(Conn) ->
 
 %% @doc This function executes the functional object Fun as a transaction.
 %% @see transaction/4
--spec transaction(connection(), fun()) -> {atomic, term()} | {aborted, term()}.
+-spec transaction(Conn, TransactionFun) -> TransactionResult
+    when Conn :: connection(),
+         TransactionFun :: fun((...) -> Result),
```

Comment:
`fun(() -> Result)` would be more exact here, since `Args = []` when left out.

Reply:
Right, I'll change it.

Reply:
Done